src/services/awards.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+import _ from 'lodash';
+import { Problem } from '@acmascis/codeforces-client/build/interfaces/problem.interface';
+import { Standings } from '@acmascis/codeforces-client/build/interfaces/standings.interface';
+import { Submission } from '@acmascis/codeforces-client/build/interfaces/submission.interface';
```

Comment:
We shouldn't be navigating the packages in this way, but this is a fault from the design of `codeforces-client` package, let's discuss this later.